src/citrine/exceptions.py

```diff
@@ -115,6 +115,12 @@ class PollingTimeoutError(NonRetryableException):
     pass
 
 
+class AccountsV3Exception(CitrineException):
```

Comment:
thinking about this again, I think we don't need this specific error, and all cases where I would use it would be better served with other, existing errors (Not implemented and similar)

Reply:
Sounds good! The simpler, the better.